requirements.txt

```diff
@@ -10,6 +10,7 @@ GitPython==3.1.40
 h5py==3.10.0
 healpy==1.16.6
 matplotlib==3.8.2
+mpi4py==3.1.6
```

Comment:
Did we decide we need MPI or not?

Reply:
"Maybe."  It went in there as an experiment (see below).  I left it in for now to facility further experiments as necessary.